### PR TITLE
Summon Guns guns up antags too.

### LINKED ISSUE #11473: SUMMON GUNS DOES NOTHING DURING WIZARD WARS
(WEB REPORT BY: kavlax REMOTE: 172.93.109.202:7777)

> Revision 4d935af35fa01c86e8925136514deebb4f59b34d 
> It doesn't make guns for wizards.

### PR DESCRIPTION
**Pull request by Allspice**
Fixes #11473

### CODE REVIEW

**Comment by Skullyton** [comment]
Doesn't this give the wizard guns too?

**Comment by Allspice** [comment]
Yep.

**Comment by ComicIronic** [comment]
The wizard is not meant to get a gun.

**Comment by SonixApache** [comment]
@ComicIronic 
The Wizard's Shotgun

**Comment by ComicIronic** [comment]
Which doesn't mean the wizard is meant to get a gun. Wizards are strong with guns. The wizard is not meant to get a gun.

**Comment by PJB3005** [comment]
@SonixApache the proc is literally called `RightAndWrong` so, yes, it's pretty clearly based on that.

**Comment by FudgePucker** [comment]
The wizard spent two points on a spell that just gave every validhunter a gun and told the entire station all at once the round type.
Giving them a gun isn't that bad when they could have just teleported to armory for zero spell points and not giving every validhunter a gun.

**Comment by gbasood** [comment]
I personally would prefer this behavior were unique to wizard wars

**Comment by clusterfack** [comment]
I'm pretty sure this is a terrible idea either way, but as @gbasood said as a minimum limit it to wizard wars if you're making this to explicitly fix the issue with wizard wars.

**Comment by Blithering** [comment]
Wizards summoning guns should get a magic gun of some description.

**Comment by Shadowmech88** [comment]
>magic gun

Like a Roulette Revolver?

**Comment by clusterfack** [comment]
You really need to add some sanity to that gun, it spawns blob cores and immovable rods

**Comment by FudgePucker** [comment]
Its a feature.

**Comment by Shadowmech88** [comment]
Given that it's only a 1/total projectiles chance per shot, I don't think it needs to be restricted as long as the projectiles exit the gun correctly.

**Comment by clusterfack** [comment]
I have never had someone manage to avoid getting meteor/immovable rod/blob core when they're trying to fuck with a testing session. There is also a huge number of variations of meteors and blob types.

**Comment by Shadowmech88** [comment]
If you feel that strongly, make a Github issue for people to discuss it, then.

File: code/game/gamemodes/wizard/rightandwrong.dm [hunk 1]
```diff
@@ -7,9 +7,7 @@
 	for(var/mob/living/carbon/human/H in player_list)
 		if(H.stat == DEAD || !(H.client))
 			continue
-		if(is_special_character(H))
```
Comment (Intigracy) [inline code/game/gamemodes/wizard/rightandwrong.dm:10]: Do me a favor and add `if(summon_type && iswizard(H))` here, keeping the continue.

Wizards shouldn't get summoned magic stuff seeing as they're intended to spend a spellpoint on it and it has a chance of giving the staves.

**Comment by ComicIronic** [comment]
Server vote or die.

**Comment by Allspice** [comment]
vote

**Pull request merged**